python/ml4ir/base/features/feature_config.py

```diff
@@ -97,19 +97,19 @@ def extract_features(self, features_dict, logger: Optional[Logger] = None):
         except KeyError:
             self.query_key = None
             if logger:
-                logger.warning("'query_key' key not found in the feature_config specified")
+                logger.warning("'%s' key not found in the feature_config specified" % FeatureConfigKey.QUERY_KEY)
```

Comment:
Here we have a class a few lines above that specifies these names, so better to not hardcode them.

Reply:
Good catch.